### DISCUSSION TITLE
folder not showing on repo

### DISCUSSION THREAD

**Discussion by chuma1409** [community/community#68053] [Repositories]
### Select Topic Area

Question

### Body

folder not showing on repo but i pushed changes

**Comment by worldwidewurf** [comment]
try adding a file in the folder this will make git reconize the folder

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
